docker-compose.yml

```diff
@@ -13,7 +13,7 @@ x-defaults: &defaults
   build:
     context: .
     args:
-      IMAGE: ${IMAGE}
+      SENTRY_IMAGE: ${SENTRY_IMAGE}
```

Comment:
If Docker Compose format haven't dropped support for defaults for variable substitution, we could have used `SENTRY_IMAGE: ${SENTRY_IMAGE:-sentry:9.1.2}`